model/src/main/java/com/basistech/rosette/apimodel/DocumentRequest.java

```diff
@@ -71,7 +68,7 @@
      */
     private final String contentType;
 
-    /**
+    /** @deprecated
```

Comment:
Should this be moved off the `/**` line?

Reply:
Ooops, good catch!